### DISCUSSION TITLE
How to see representation_model topics

### DISCUSSION THREAD

**Discussion by brianferrell787** [MaartenGr/BERTopic#1017] [General]
Hello, 

I see we can use openai, cohere, etc to change how the topic labels are written, like below:
![image](https://user-images.githubusercontent.com/58923369/219884384-f9e4d66f-fe08-4b90-a0dd-63a71a5a7840.png)

But how do we see this in the code?
When I run get topic info, I still see the words with "_" spacing. 
![image](https://user-images.githubusercontent.com/58923369/219884410-700914a3-4ad9-498f-aa9f-d9aafd7369ee.png)

**Comment by MaartenGr** [comment]
This might be related to your answer [here](https://github.com/MaartenGr/BERTopic/issues/1011#issuecomment-1435735463) but it is important to follow along with the guide [here](https://maartengr.github.io/BERTopic/getting_started/representation/representation.html#openai) for using the openai package within BERTopic. You will need to create an `OpenAI` model from within BERTopic (`bertopic.representation_model.OpenAI`).
